src/ast/ast_validations.c

```diff
@@ -12,6 +12,7 @@
 #include "../arithmetic/repository.h"
 #include "../arithmetic/arithmetic_expression.h"
 #include <assert.h>
+#include "../query_ctx.h"
```

Comment:
I don't think this include is necessary.

Reply:
removed